src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/Example1.java

```diff
@@ -0,0 +1,37 @@
+/*
+EmptyLineSeparator
+allowNoEmptyLineBetweenFields = (default)false
+allowMultipleEmptyLines = (default)true
```

Comment:
Do we have validation on defaults? Example: if we change the default value of this property to false, will we fail somewhere? Otherwise, I do not see the value in having a bunch of default property values in these configs. This seems analogous to the extraneous `// ok` comments; it is just noise.

cc @romani @rnveach

Reply:
> Example: if we change the default value of this property to false, will we fail somewhere?

To be clear, default value of the property refers to the actual default value defined in the check.

Reply:
@nrmancuso Done. Removed `(default)` properties.